spine-ts/spine-player/src/Player.ts

```diff
@@ -239,6 +247,7 @@ export class SpinePlayer implements Disposable {
 <button class="spine-player-button spine-player-button-icon-skins"></button>
 <button class="spine-player-button spine-player-button-icon-settings"></button>
 <button class="spine-player-button spine-player-button-icon-fullscreen"></button>
+<button class="spine-player-button spine-player-button-icon-audio-off"></button>
```

Comment:
See discussion in the comments section of this PR. TL;DR: only show the button if `audioFolderUrl` is given.